MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include setup.py README.md MANIFEST.in LICENSE AUTHORS
 recursive-include ./ requirements*.txt
+recursive-include ./config/cdc *
 recursive-include ./config/relay *
```

Comment:
2 things:

1. We should do `graft ./config` at this point
2. These files are not needed for prod at all but we are bundling them into our wheel and prod images only for test and dev flows. We should find a better way to do this @billyvg

Reply:
Hmm, we could copy them over in our test container -- do we know if `relay` was added purely for test workflows?

Reply:
@RaduW  (about relay) ?

Reply:
@billyvg @BYK  how confident are you the issue is with MANIFEST.in. I tried to build the sentry package locally and the whole config directory is in the resulting `dist`. (without adding anything to MANIFEST).

Reply:
I'm not too confident - you can see the snuba PR here https://github.com/getsentry/getsentry/pull/5397 - it's hard to tell, but snuba is starting and immediately crashing due to the config file not being found. It could be an issue with the docker test image in getsentry.

If we're not immediately using CDC to test, we can revert the `MANIFEST` changes until we need it

Reply:
ok. 
Well I won't remove the change as it won;t hurt (the fact relay config is mentioned there makes me think the cdc one should be there as well).

Reply:
The relay config files are indeed only useful for testing.

@billyvg If we add them as part of the test container we would have to use a dedicated relay docker image 
that is the production docker image + the config dir (since I think it would be a bad idea to have
a production container with backed-in credentials). 

If we really dislike just having them in sentry we could generate them as part of our sentry dev setup.
A credentials file can be generated by invoking relay with the option `--credentials`. 
We would have to copy the `public_key` from the credentials file into the user's  `~\.sentry\sentry-conf.py`
like so: `SENTRY_RELAY_WHITELIST_PK = [ <The key> ]`.
All of these are doable with a bit of effort, the question is if it is worth it ?

Since they are only 2 tiny files I think we should just include them and simplify our lives.
either `graft` or `recurisve-inclue *` work for me (graft is nicer in this case).

Reply:
This change seems to me beyond the scope of this PR (like changing how relay configuration is provided in dev).
Does anybody have concerns with merging this? I can copy the cdc config in the test container itself (leaving relay where it is). @billyvg In such case (copying cdc config in the test container) do you know if there is a way to test that locally ?

Reply:
I think this is fine to merge - testing it locally might be a bit of a pain. you'd have to build the `getsentry/Dockerfile.test` image and run it.